walkthroughs/week-3-server/examples/todo-list/src/main/java/com/google/sps/servlets/NewTaskServlet.java

```diff
@@ -35,7 +35,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     Entity taskEntity = new Entity("Task");
     taskEntity.setProperty("title", title);
     taskEntity.setProperty("timestamp", timestamp);
-
+    `     
```

Comment:
this seems unintentional

Reply:
Yes, it was. As at now, I can't seem to get the appropriate command to undo it.